.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: python
 python:
-  - "3.6"      # current detault Python on Travis CI
   - "3.7"
   - "3.8"
   - "3.9"
-  - "3.10-dev"  # 3.10 development branch
+  - "3.10"
+  - "3.11"
+  - "3.11-dev"
```

Comment:
If adding a `-dev` version, it should probably be 3.12.  I'm not sure that Travis is actually used at all anymore, though; if not, this file should probably just be removed as it's already out of date (3.6 was already dropped, `buildout` has been dropped in favor of `poetry`).